### PR TITLE
Added Hindi translation

### PR DESCRIPTION
**Pull request by gostsdmitry**
Added Hindi translation

### CODE REVIEW

File: po/hi.po [hunk 1]
```diff
@@ -0,0 +1,1546 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
```
Comment (multani) [inline po/hi.po:5]: Can you update the header with relevant information?

**Review by multani — APPROVED** [review]
Thanks @gostsdmitry! I'll merge this once the header is up to date.

I'll update the POT as well (it's a bit old) and I'll post some more information about translations (I have a project setup on Transifex, but I realized it was never announced anywhere...)

File: po/hi.po (hunk 1 above)
Comment (gostsdmitry) [reply po/hi.po]: Do I have to indicate my E-Mail in item #4?

Comment (gostsdmitry) [reply po/hi.po]: I mean the fourth line of the translation file.

Comment (multani) [reply po/hi.po]: It's not mandatory, but it helps if another translator wants to get in touch in the future, I suppose.

Comment (gostsdmitry) [reply po/hi.po]: > It's not mandatory, but it helps if another translator wants to get in touch in the future, I suppose.

added E-mail.

**Review by multani — APPROVED** [review]
Thanks a lot @gostsdmitry!

**Pull request merged**